test/event_test.js

```diff
@@ -1,221 +1,108 @@
-suite('event', function() {
-  var Promise     = require('promise');
-  var launch      = require('../bin/server');
-  var SockJS      = require('sockjs-client-node');
-  var assert      = require('assert');
-  var slugid      = require('slugid');
-  var taskcluster = require('taskcluster-client');
-  var debug       = require('debug')('test:event');
-  var base        = require('taskcluster-base');
+'use strict'
```

Comment:
Since we're transpiling with babel, we don't really want to specify `use strict` here, right? The transpiler will handle it for us.